faros_event.sh

```diff
@@ -884,6 +721,24 @@ function resolveDeployInput() {
     if ! [ -z "$deploy_end_time" ]; then
         deploy_end_time=$(convert_to_iso8601 "$deploy_end_time")
     fi
+    if ! [ -z "$deploy_app_tags" ]; then
+        tags_regex="^[^:]+:[^,]+(,[^:]+:[^,]+)*$"
+        if [[ "$deploy_app_tags" =~ $tags_regex ]]; then
+            IFS=',' read -ra deploy_app_tag_arr <<< "$deploy_app_tags"
+        else
+            err "deploy_app_tags could not be parsed"
+            fail
+        fi
+    fi
+    if ! [ -z "$deploy_app_paths" ]; then
+        paths_regex="^[^,]+(,[^,]+)*$"
+        if [[ "$deploy_app_paths" =~ $paths_regex ]]; then
+            IFS=',' read -ra deploy_app_path_arr <<< "$deploy_app_paths"
+        else
+            err "deploy_app_paths could not be parsed"
+            fail
+        fi
+    fi
```

Comment:
Can we instead allow sending application tags & paths as strings and parse in `/events` API and error out of fails?

Reply:
We did something similar before already, where we allowed passing either an array or a string for some argument.

Reply:
Sounds good! The change is [here](https://github.com/faros-ai/clio/pull/471)